GCP/DirectoryInsights/README.md

```diff
@@ -12,20 +12,78 @@ _Note: This document assumes the use of Python 3.9_
 
 ## Pre-requisites
 - [Your JumpCloud API key](https://docs.jumpcloud.com/2.0/authentication-and-authorization/authentication-and-authorization-overview)
+- Google Cloud Admin/Owner account with these roles:
+  - ```roles/serviceusage.serviceUsageAdmin```
+  - ```roles/cloudbuild.builds.editor```
+  - ```roles/resourcemanager.projects.setIamPolicy```
 - [GCLOUD CLI installed](https://cloud.google.com/sdk/docs/install)
-- [Google Cloud Build](https://cloud.google.com/build/docs/securing-builds/configure-access-for-cloud-build-service-account)
-  - Permissions:
-    - Cloud Functions = Enabled
-    - Cloud Scheduler = Enabled
-    - Secret Manager = Enabled
-  - Cloud Build Service account email: admin access to services above
-- JumpCloud administrator will need `Cloud Build Editor` role with their GCP account
-- Cloud Functions Invoker service account
+  - After installing the CLI, run ```gcloud auth login ``` and login with your Admin/Owner account
+- On your CLI, run these commands to enable the [services](https://cloud.google.com/apis?hl=en) needed to build the app:
```

Comment:
In my opinion all of these commands need to be code blocks not individual lines of code. It's **really** painful to copy these line by line and the customers will not read these anyways, they are just going to copy and paste what's written.

Reply:
Put the commands in code blocks :)